src/shared/config/apiUrls.ts

```diff
@@ -109,8 +109,19 @@ const apiUrls = {
       query: { query: `accession:${accession}` },
     }),
   idMappingFields: joinUrl(apiPrefix, '/configure/idmapping/fields'),
-  entry: (id: string | undefined, namespace: Namespace) =>
-    id && joinUrl(apiPrefix, namespace, id),
+  entry: (id: string | undefined, namespace: Namespace, columns?: Column[]) => {
+    if (!id) {
+      return undefined;
+    }
+    const url = joinUrl(apiPrefix, namespace, id);
+    if (columns?.length) {
+      return queryString.stringifyUrl({
+        url,
+        query: { fields: columns.join(',') },
+      });
+    }
```

Comment:
Is this for future-proofing for download customisation on entry pages?
I couldn't find the columns parameter used anywhere in the PR or in the rest of the codebase

Reply:
Ah, I see, makes sense.
I guess leave it here because it might be useful in the future anyway